ci-operator/config/openshift/release/openshift-release-master__nightly-4.18.yaml

```diff
@@ -469,7 +469,9 @@ tests:
     env:
       TEST_ARGS: --disable-monitor=service-type-load-balancer-availability
       TEST_SKIPS: provisioning should mount multiple PV pointing to the same storage
```

Comment:
To improve the management, I wonder if we can consolidate those envs in the workflow `upi-vsphere-platform-external-ccm` as we'll need this in all jobs using it, wdyt?

Reply:
For this removal, its only failing in 4.18 at the moment.  I can look into moving some of the other args as well if it makes sense.  Let me investigate and i'll return with my findings.  Thanks!

Reply:
I'll do a separate PR on moving these.  It should be fine to skip the tests in older OCP version tests.